<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="LAPTOP-B81G2UPP" failures="1" tests="4" name="Test" time="9.902" errors="0" timestamp="05 Mar 2022 11:25:27 GMT">
  <testcase classname="com.aspire.test.ManufacturingFeature_Test" name="createInventory" time="0.031">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.aspire.page.LoginPage.checkLoginPage(LoginPage.java:33)
at com.aspire.page.LoginPage.login(LoginPage.java:36)
at com.aspire.test.ManufacturingFeature_Test.loginFunc(ManufacturingFeature_Test.java:28)
at com.aspire.test.ManufacturingFeature_Test.createInventory(ManufacturingFeature_Test.java:18)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- createInventory -->
  <testcase classname="com.aspire.test.LoginFeature_Test" name="inValidLogin" time="0.751"/>
  <testcase classname="com.aspire.test.LoginFeature_Test" name="validLogin" time="6.432"/>
  <testcase classname="com.aspire.test.LoginFeature_Test" name="validLoginLogout" time="2.702"/>
  <testcase classname="com.aspire.baseUtil.BaseTest" name="@BeforeSuite initTest" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite initTest -->
  <testcase classname="com.aspire.baseUtil.BaseTest" name="@AfterSuite tearDownTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDownTest -->
</testsuite> <!-- Test -->
